src/vss-extension.json

```diff
@@ -58,13 +58,14 @@
     "type": "git",
     "uri": "https://github.com/deadlydog/AzureDevOps.WindowsScheduledTasks"
 	},
-	"badges": [{
-			"href": "https://dev.azure.com/deadlydog/OpenSource/_build/latest?definitionId=19&branchName=master",
+	"badges": [
+		{
+			"href": "https://dev.azure.com/deadlydog/OpenSource/_build/latest?definitionId=19\u0026branchName=master",
 			"uri": "https://dev.azure.com/deadlydog/OpenSource/_apis/build/status/AzureDevOps.WindowsScheduledTasks?branchName=master",
 			"description": "Azure DevOps build for the project"
 		},
 		{
-			"href": "https://dev.azure.com/deadlydog/OpenSource/_release?view=all&definitionId=1",
+			"href": "https://dev.azure.com/deadlydog/OpenSource/_release?view=all\u0026definitionId=1",
```

Comment:
Why are you using the unicode for '&' ?

Reply:
It auto-converted it to unicode when running the tool to pack up the extension as a .vsix file.